teamA/crud-express/heeyeon/routes/likesRouter.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const router = express.Router();
+
+const likesController = require('../controllers/likesController');
+router.post('', likesController.pressLikes);

```

Comment:
- 해당 코드도 가독성을 위해 한 줄 띄어 수정해주세요 !
```suggestion
const likesController = require('../controllers/likesController');

router.post('', likesController.pressLikes);
```